queue.py

```diff
@@ -0,0 +1,63 @@
+from dynamic_array import DynamicArray
+
+class Queue:
+    def __init__(self, size):
+        self.size = size
+        self.array = DynamicArray()
```

Comment:
Your size isn't size you initiate DynamicArray  with ! Which internally intialises with 10! This can lead to incorrect state for L11 to L14
```
 if self.right - self.left == self.size:
            self.right = 0
            self.left = 0
```

Reply:
fixed